app/scripts/views/mixins/password-prompt-mixin.js

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * FormView plugin to convert a placeholder into a
+ * floating label if the input changes. Behavior is automatically
+ * hooked up in `afterRender`
+ */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var $ = require('jquery');
+  var Tooltip = require('views/tooltip');
+  // does this belong here, or in lib/constants
+  var TOOLTIP_MESSAGES = {
+    FOCUS_PROMPT_MESSAGE: '8 characters minimum, but longer if you plan to sync passwords.',
+    INITIAL_PROMPT_MESSAGE: 'A strong, unique password will keep your Firefox data safe from intruders.',
+    MORE_INFO_LINK: ' <a href="">More info..</a>',
+    WARNING_PROMPT_MESSAGE: 'This is a common password; please consider another one.'
+  };
+
+  module.exports = {
```

Comment:
I don't understand this, all the other mixins (sign_up, sign_in, floating_placeholder, etc.) use this. What am I missing?